src/module.ts

```diff
@@ -350,10 +351,15 @@ export default defineNuxtModule<ModuleOptions>({
 
         // Output the generated documents if desired.
         if (options.outputDocuments) {
-          const destFolder = resolve(
-            nuxt.options.buildDir,
-            'nuxt-graphql-middleware/documents',
-          )
+          let destFolder
```

Comment:
Alternatively you could move this to a separate method called `getOutputDocumentsPath` that returns the resolved output path or null and here just call it like this:

```typescript
const outputDocumentsPath = getOutputDocumentsPath()
if (outputDocumentsPath) {
  outputDocuments(outputDocumentsPath, documents)
}
```

Reply:
Done